### ISSUE TITLE
Lhost issue

### ISSUE THREAD

**Issue by peterpt** [tiagorlampert/CHAOS#4]
Script does not allow ip 1.1.1.1

**Comment by peterpt** [comment]
Check Pull requests for fix , or change it yourself and close this issue

**Comment by tiagorlampert** [comment]
Hi, thanks for fix it.

**Comment by peterpt** [comment]
No problem ;) , i notice that bug because i was testing your tool here and my network lan runs over 1.1.1.1 dhcp relay .
And since i notice that if user inputed 25555.5555.555.5555 the Lhost also accept it , then i made the change here to not allow more than 15 characters (255.255.255.255) .

I also have a another idea to you implement in in your CHAOS git , i already implemented it here in my computer , but did not updated my Chaos git with it yet .
Is creating a bash script to start chaos , this way is more simple for the user to start the script on Chaos directory with "./start" or "./chaos" , than remember in future "go run CHAOS.go" .
basically what the bash script have is that "go" start command inside .

**Comment by tiagorlampert** [comment]
I'll compile it, can be executed using ./CHAOS.
I have a [issue](https://github.com/tiagorlampert/CHAOS/issues/3) opened, then can be solved using that.

Thanks for idea!

**Comment by peterpt** [comment]
Compiled version works perfectly on Devuan Linux .
Nice tool :))

I have other idea but i will create a new issue and close this one .
